CHANGELOG_ANDROID.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## [1.39.111](https://github.com/brave/brave-browser/releases/tag/v1.39.111)
+
+ - Added Brave Firewall + VPN. ([#12197](https://github.com/brave/brave-browser/issues/12197))
+ - Added support for blob partitioning. ([#21746](https://github.com/brave/brave-browser/issues/21746))
+ - Implemented eth_getEncryptionPublicKey for Brave Wallet. ([#19276](https://github.com/brave/brave-browser/issues/19276))
```

Comment:
Can probably reword this one as Implemented `Get Public Encryption Key` for Brave Wallet